__tests__/app.test.js

```diff
@@ -164,3 +164,58 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: Responds with the posted comment", () => {
+    const newComment = {
+      username: "rogersop",
+      body: "The sixth sick sheik's sixth sheep's sick",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body: { comment } }) => {
+        console.log({ body: { comment } })
+        console.log(comment)
+        expect(comment).toMatchObject({
+          body: "The sixth sick sheik's sixth sheep's sick",
+          votes: expect.any(Number),
+          author: "rogersop",
+          article_id: 1,
+          created_at: expect.any(String),
```

Comment:
there should also be a comment_id, indicating that it has been added to the database!